Source/SLSC Switch/Help/HTML Help Source/Main.htm

```diff
@@ -14,13 +14,44 @@
 		<script src="common.js" type="text/javascript"></script><script src="js/expandable_tree.js" type="text/javascript"></script>
 		<link rel="STYLESHEET" type="text/css" href="css/expandable_tree.css">
 		<script src="launchhelp.js" type="text/javascript"></script><script src="js/dynamicjumps.js" type="text/javascript"></script>
+		<style type="text/css">
+			.auto-style1 {
+				color: red;
+			}
+		</style>
 	</head>
 	<body>
 		<noscript>
 			<p class="Body">JavaScript is disabled. <a href="veristandmerge.chm::/JavaScript_Disabled.html">Details</a></p>
 			<hr width="100%" noshade>
 		</noscript>
 		<h1>SLSC Switch</h1>
-		Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</br>
+		<p class="Body">The <strong>SLSC Switch Custom Device</strong> is a VeriStand add-on that supports SLSC routing modules.</p>
```

Comment:
> `The <strong>SLSC Switch Custom Device</strong> is a VeriStand add-on`

This feels unnecessary in this context. We are showing the help in VeriStand.

Reply:
Most of our custom devices start with this rather obvious introduction.

- The **Scan Engine and EtherCAT** custom device allows you to communicate with I/O in local and remote chassis using the NI Scan Engine.
- This **SLSC EDS Addon** provides the ability to monitor and communicate with any SLSC Module inside a SLSC Chassis in a generic way.
- This **SLSC-12201 Module Addon** provides the ability to monitor and communicate with SLSC-12201 Digital Input/Output Module in a SLSC Chassis.

We can omit "VeriStand" and maybe be consist with our spelling of addon(add-on?)(add on?).